website/docs/r/servicecatalog_budget_resource_association.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+subcategory: "Service Catalog"
+layout: "aws"
+page_title: "AWS: aws_servicecatalog_budget_resource_association"
+description: |-
+  Manages a Service Catalog Budget Resource Association
+---
+
+# Resource: aws_servicecatalog_budget_resource_association
+
+Manages a Service Catalog Budget Resource Association.
+
+-> A "resource" is either a Service Catalog portfolio or product.
```

Comment:
See [this comment](https://github.com/hashicorp/terraform-provider-aws/pull/19448#discussion_r636105972).